### PR TITLE
docs: update firecracker for new home of tc-redirect-tap plugin

### PR DESCRIPTION
**Pull request by smira**
See https://github.com/firecracker-microvm/firecracker-go-sdk/issues/174#issuecomment-655798205

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/2292)

### CODE REVIEW

**Comment by smira** [comment]
added a section on `make kernel initrafms`, as by default kernel is not built

File: docs/website/content/v0.6/en/guides/local/firecracker.md [hunk 1]
```diff
@@ -75,15 +75,25 @@ sudo cp cni-plugins-linux/{bridge,firewall,static} /opt/cni/bin
...
+> Note: if `$GOPATH` is not set, it defaults to `~/go`.
+
+## Build Talos kernel and initramfs
+
+Firecracker provisioner depends on Talos kernel and initramfs:
+
+```bash
+make kernel initramfs
+```
+
```
Comment (andrewrynhard) [inline docs/website/content/v0.6/en/guides/local/firecracker.md:96]: An alternative is that they could also download these from a release.

Comment (smira) [reply docs/website/content/v0.6/en/guides/local/firecracker.md:96]: good point, let me add that, guide doesn't assume you have to be a developer, thanks!

**Pull request merged**